browser/src/control/Control.NotebookbarImpress.js

```diff
@@ -199,7 +199,7 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 				'enabled': 'true',
 				'children': [
 					{
-						'id': 'downloadas2',
+						'id': 'downloadas',
 						'type': 'menubartoolitem',
 						'text': _('Download as'),
 						'command': '.uno:InsertGraphic'
```

Comment:
just noticed: what it does here? InsertGraphic command in download as?

Reply:
exactly , that's why I really would like to re-implement this part, I think Alexandru just use a random command as a placeholder